### ISSUE TITLE
NetworkX generated .gexf files are not parsed/processed correctly

### ISSUE THREAD

**Issue by abitofalchemy** [jacomyal/sigma.js#846]
Great project!
I am trying to generate a .gexf file from networkx and substitute it in where a sample project `examples/data/arctic.gexf`, but that doesn't work out of the box. Is there something I could to fix this or make this better? Should I read in the edgelist in Gephi and then output as .gexf instead?

**Comment by Yomguithereal** [comment]
How is your graph rendered? I don't see anything problematic with your gexf file's head just yet.

**Comment by jackfischer** [comment]
I'm having the same problem; @abitofalchemy have you figured out a solution?

**Comment by abitofalchemy** [comment]
No, but I need to get back to this. It's a great promising framework for visualization, but I do a lot of my work with networkx and igraph.

**Comment by AVermeij** [comment]
@jackfischer @abitofalchemy, try checking your console and locating the lines in your GEXF which lead to parse issues? There may be some character encodings in your files which stir things up. Also see https://github.com/jacomyal/sigma.js/issues/700

**Comment by darie2808** [comment]
I have the same issue :(
